<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>792</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1380</width>
    <height>920</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>1380</width>
     <height>920</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>125</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QSpinBox" name="spinBoxRed">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>128</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxGreen">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxBlue">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>42</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxRedM">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>40</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxGreenM">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>70</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxBlueM">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>100</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Red Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Green Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Blue Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>10</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Zoom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>70</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Zoom Y</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxZoomX">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>40</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1280</number>
       </property>
       <property name="value">
        <number>400</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Zoom X</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxZoomY">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>70</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>720</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxScale">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>100</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>100</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>20</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>60</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>100</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonReset_2">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>90</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxMaxIt">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>100</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>100</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max iterations</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxWidth">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>40</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1920</number>
       </property>
       <property name="value">
        <number>800</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Height (pixels)</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxHeight">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>70</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1080</number>
       </property>
       <property name="value">
        <number>400</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Width (pixels)</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelImage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1280</width>
        <height>720</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
